packer.json

```diff
@@ -21,24 +21,21 @@
       "source_ami_filter": {
         "filters": {
           "virtualization-type": "hvm",
-          "name": "ubuntu/images/*ubuntu-xenial-16.04-amd64-server-*",
+          "name": "ubuntu/images/*ubuntu-bionic-18.04-amd64-server-*",
           "root-device-type": "ebs"
         },
         "owners": ["099720109477"],
         "most_recent": true
       },
       "instance_type": "t3.micro",
       "ssh_username": "ubuntu",
-      "ami_name": "elotl-kipdev-{{user `version`}}",
-      "ami_groups": [
-          "all"
-      ]
```

Comment:
If you remove `ami_groups: [all]`, that will make the images we build private, thus anyone outside of our account won't be able to use them.

Reply:
:+1: reverted. We probably need to think how to workaround this encryption error.